docs/en/loading/Kafka-connector-starrocks.md

```diff
@@ -11,99 +11,173 @@ The Kafka connector can seamlessly integrate with Kafka Connect, which allows St
 
-- Confluent cloud:
+- Confluent Cloud:
 
-  > **NOTE**
-  >
-  > The Kafka connector is not currently uploaded to Confluent Hub. You need to upload the compressed file to Confluent cloud.
+  Currently, the Kafka connector is not uploaded to Confluent Hub. You need to download and extract [starrocks-kafka-connector-xxx.tar.gz](https://github.com/StarRocks/starrocks-connector-for-kafka/releases), package it into a zip file and upload the zip file to Confluent Cloud.
 
-### Create StarRocks table
+## Usage
 
-Create a table or tables in StarRocks according to Kafka Topics and data.
+This section uses a self-managed Kafka cluster as an example to explain how to configure the Kafka connector and the Kafka Connect, and then run the Kafka Connect to load data into StarRocks.
 
-## Examples
+### Prepare a dataset
 
-The following steps take a self-managed Kafka cluster as an example to demonstrate how to configure the Kafka connector and start the Kafka Connect (no need to restart the Kafka service) in order to load data into StarRocks.
+Suppose that JSON-format data exists in the topic `test` in a Kafka cluster.
 
-1. Create a Kafka connector configuration file named **connect-StarRocks-sink.properties** and configure the  parameters. For detailed information about parameters, see [Parameters](#parameters).
+```JSON
+{"id":1,"city":"New York"}
+{"id":2,"city":"Los Angeles"}
+{"id":3,"city":"Chicago"}
+```
+
+### Create a table
+
+Create the table `test_tbl` in the database `example_db` in the StarRocks cluster according to the keys of the JSON-format data, .
+
+```SQL
+CREATE DATABASE example_db;
+USE example_db;
+CREATE TABLE test_tbl (id INT, city STRING);
+```
+
+### Configure Kafka connector and Kafka Connect, and then run Kafka Connect to load data
 
-    ```Properties
+#### Run Kafka Connect in standalone mode
+
+1. Configure the Kafka connector. In the **config** directory under the Kafka installation directory, create the configuration file **connect-StarRocks-sink.properties** for the Kafka connector, and configure the following parameters. For more parameters and dsescriptions, see [Parameters](#Parameters).
+
+    ```yaml
     name=starrocks-kafka-connector
     connector.class=com.starrocks.connector.kafka.StarRocksSinkConnector
-    topics=dbserver1.inventory.customers
-    starrocks.http.url=192.168.xxx.xxx:8030,192.168.xxx.xxx:8030
-    starrocks.username=root
+    topics=test
+    key.converter=org.apache.kafka.connect.json.JsonConverter
+    value.converter=org.apache.kafka.connect.json.JsonConverter
+    key.converter.schemas.enable=true
+    value.converter.schemas.enable=false
+    # The HTTP URL of the FE in your StarRocks cluster. The default port is 8030.
+    starrocks.http.url=192.168.xxx.xxx:8030
+    # If the Kafka topic name is different from the StarRocks table name, you need to configure the mapping relationship between them.
+    starrocks.topic2table.map=test:test_tbl
+    # StarRocks username
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/dabb39b1-5522-45cc-8793-8cbfe804440f)
1. -> Enter the StarRocks username.
2. -> Enter the StarRocks password.
Also, what account do the StarRocks username and password belong? A better and clearer way to describe these two parameters should be "Enter the username of the xxx account" and "Enter the password of the xxx/preceding account".